src/components/Scorecards/ScorecardDetailsPage/ScorecardMetadataCard/ScorecardMetadataCard.tsx

```diff
@@ -69,7 +70,11 @@ export const ScorecardMetadataCard = ({
         {scorecard.description && (
           <Box mb={2} className={classes.markdownBox}>
             <CaptionTypography variant="caption">Description</CaptionTypography>
-            <MarkdownContent content={scorecard.description} />
+            <Truncated
+              text={scorecard.description || ''}
```

Comment:
nit: prefer null coalescing for semantics
```suggestion
              text={scorecard.description ?? ''}
```